.github/workflows/test.yml

```diff
@@ -25,21 +33,25 @@ jobs:
       - uses: ./.github/actions/setup
       - run: yarn prepare
       - run: yarn test
-
-  cypress-build:
+  
+  build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup
       - run: yarn prepare
-
       - run: yarn build
       - uses: actions/upload-artifact@v2
         with:
           name: build
           path: build
           if-no-files-found: error
 
+  cypress-build:
```

Comment:
why does the cypress build need to change?

Reply:
it doesn't, I was just prepping for another test for deps size.